test/unit/features/options/methods.spec.js

```diff
@@ -24,4 +24,11 @@ describe('Options methods', () => {
     })
     expect(`method "hello" has an undefined value in the component definition`).toHaveBeenWarned()
   })
+
+  it('should warn non object', () => {
```

Comment:
These tests can also be refactored into a function. It should also test it doesn't print the warning in the right scenario